lib/model/config.dart

```diff
@@ -149,34 +154,35 @@ class Config {
   }
 
   /// Azure AD OAuth Configuration. Look at individual fields for description.
-  Config({
-    required this.tenant,
-    this.policy,
-    required this.clientId,
-    this.responseType = 'code',
-    String? redirectUri,
-    required this.scope,
-    this.webUseRedirect = false,
-    this.responseMode,
-    this.state,
-    this.prompt,
-    this.codeChallenge,
-    this.codeChallengeMethod,
-    this.nonce = '12345',
-    this.tokenIdentifier = 'Token',
-    this.clientSecret,
-    this.resource,
-    this.isB2C = false,
-    this.loginHint,
-    this.domainHint,
-    this.codeVerifier,
-    this.userAgent,
-    this.isStub = false,
-    this.loader = const SizedBox(),
-    AndroidOptions? aOptions,
-    required this.navigatorKey,
-    this.origin,
-  })  : authorizationUrl = isB2C
+  Config(
+      {required this.tenant,
```

Comment:
This doesn't look like it will conform to standard dart fmt. This costs the package "pub points" - please ensure you have reformatted (IDE should do it automatically for you - ctrl-alt-f in vscode for example).

Reply:
I tried the format command but nothing changes with the layout. Can you format it and send me a screenshot of what the differences are?

Reply:
Sorry, i gave the wrong keystrokes for vscode - it's SHIFT-ALT-F, not CTRL-ALT-F. Wrote that without a PC handy to check sorry.

Before:
![image](https://user-images.githubusercontent.com/16790181/216730124-1d5135d7-92af-4d6c-8dae-a9b51c279eca.png)

After:
![image](https://user-images.githubusercontent.com/16790181/216730103-4009c9fe-6eb4-45e3-a379-2395f6cc24ef.png)

Reply:
No problem, I tried all the key combinations anyway. If I format it as shown in After and hit SHIFT-ALT-F it formats it to the Before view. As it turns out Dart formatter is the default for dart files in my VS Code settings. So even just saving, it formats it as described. I don't see how to get around this unless you just format it and submit a commit to lock that in on your branch.

Reply:
I suspect setting the default for customParameters which has curly braces cause the formatting to stick to one line.

Reply:
It's probably because there's no trailing comma after the last parameter.